source/sky_triangle/skytriangle.cpp

```diff
@@ -173,6 +173,11 @@ void SkyTriangle::resize(int w, int h)
     m_height = h;
 }
 
+void SkyTriangle::resize(GLFWwindow * window)
+{
+    glfwGetFramebufferSize(window, &m_width, &m_height);
```

Comment:
Neat solution, but I'm not sure if we will accept this for all of our examples, demos, and tools.

Our alternative is registering a framebuffer resize event with glfw (instead of a window resize event) and passing the new parameters to the examples/painters. Is there a special reason why you use this?